LibOS/shim/src/shim_async.c

```diff
@@ -108,7 +108,7 @@ int install_async_event (PAL_HANDLE object, unsigned long time,
     
     unlock(async_helper_lock);
```

Comment:
This line should go away - otherwise we are double-unlocking the async_helper_lock at line 114.  Perhaps this is a merge error?

Seems likely.

Reply:
You are right, it is a merge error. I will fix it right away.